runtime/services/catalog/migrator/sources/sources.go

```diff
@@ -300,7 +300,7 @@ func mergeFromParsedQuery(apiSource *runtimev1.Source, env map[string]string, re
 		if err != nil {
 			return err
 		}
-	case "s3", "gcs":
+	case "s3", "gcs", "azure":
 		apiSource.Connector = c
 		props["path"] = p
```

Comment:
I think similar case handling for `azure` is needed these places:

- Line 71 in `runtime/drivers/duckdb/transporter/duckdb_to_duckdb.go`
- Line 170 in `runtime/connections.go`